src/CPULSSolver.cpp

```diff
@@ -313,28 +298,41 @@ void CPULSSolver::computeFSRSources(int iteration) {
         src_y = scatter_source_y + chi[g] * fission_source_y;
```

Comment:
For each source region, the total fission source is independent on the group index g. So line 286-295 should be put ahead to line 236 for efficiency consideration.